shuttle/src/runtime/thread/continuation.rs

```diff
@@ -255,8 +257,20 @@ impl std::fmt::Debug for PooledContinuation {
 unsafe impl Send for PooledContinuation {}
 
 /// Possibly yield back to the executor to perform a context switch.
-pub(crate) fn switch() {
+/// This function should be called *before* any visible operation.
+/// If each visible operation has a scheduling point before it, then there will
+/// be a potential context switch *in between* each visible operation, which
+/// is a necessary condition for Shuttle's completeness.
+///
+/// Putting scheduling points before visible operations, rather than after, has the
+/// advantage of giving the scheduling algorithm *maximum information* to make scheduling
```

Comment:
nit: replace `maximum information` -> `additional information`

(there's no reasonable metric by which this is `maximum`)